jsHelper/spicetifyWrapper.js

```diff
@@ -909,6 +909,14 @@ class _HTMLGenericModal extends HTMLElement {
         this.querySelector("button").onclick = this.hide.bind(this);
         const main = this.querySelector("main");
 
+        const toCheck = this.querySelector('.GenericModal');
```

Comment:
Can you remove this `const` declaration since it's only used once?

Reply:
Sure :)
I have added another commit.